pip_audit/cli.py

```diff
@@ -147,6 +164,12 @@ def audit():
         type=Path,
         help="the directory to use as an HTTP cache for PyPI; uses the `pip` HTTP cache by default",
     )
+    parser.add_argument(
```

Comment:
I decided to stick with a similar interface to `pip` instead of having a `--no-progress` flag.